pkg/monitor/worker.go

```diff
@@ -80,6 +97,21 @@ func (mon *monitor) changefeed(ctx context.Context, baseLog *logrus.Entry, stop
 							fps == api.ProvisioningStateDeleting):
 					mon.deleteDoc(doc)
 				default:
+					// check if we have a Hive shard config and if not start the recursive auth call
+					// in the future we will have the shard index set on the api.OpenShiftClusterDocument
+					// but for now we simply select Hive (AKS) shard 1
+					// e.g. shard := mon.hiveShardConfigs[doc.shardIndex]
+					shard := 1
```

Comment:
seems to be somewhat duplicated with line 233

Reply:
Intentionally duplicated. See the comment below.